### PR TITLE
Webscraping with Beautifulsoup

### PR DESCRIPTION
**Pull request by vaishali-sharma-20**
Uploading blogs on Web Scraping with Beautifulsoup.

## Related Issue
Issue: #399

## Description
This task is assigned to me for technical writing of web scraping with Beautifulsoup using Python.
I have created two files as
1. Step-by-step guide for Web Scraping
2. Example
 I have saved these files in a new folder named Web Scraping with Beautiful soup.

## Type of PR

- [x] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
I will upload images that I used in blogs.

### CODE REVIEW

**Review by CSEdgeOff — CHANGES_REQUESTED** [review]
Even the Titles are changed 
Make proper changes and add a Banner images and also make sure of SEO

**Review by vaishali-sharma-20 — COMMENTED** [review]
Image updated.  Let me know if any more changes are needed.

File: posts/Web Scraping tool with BeautifulSoup/WebScrapingWithBeautifulSoup.html
Comment (Durgesh-Vaigandla) [inline posts/Web Scraping tool with BeautifulSoup/WebScrapingWithBeautifulSoup.html:1]: Title was not correct

File: posts/Web Scraping tool with BeautifulSoup/ExampleOfBuildingWebScrapingApplication.html
Comment (Durgesh-Vaigandla) [inline posts/Web Scraping tool with BeautifulSoup/ExampleOfBuildingWebScrapingApplication.html:1]: Title was not correct

Comment (vaishali-sharma-20) [reply posts/Web Scraping tool with BeautifulSoup/ExampleOfBuildingWebScrapingApplication.html:1]: Please check it once again. I have changed the title as well as some meta data.

**Comment by vaishali-sharma-20** [comment]
Hello @Durgesh-Vaigandla and @CSEdgeOff, I have updated the title and made all the relevant changes as discussed. Could you please review the updates and let me know if everything is in order?

**Comment by Durgesh-Vaigandla** [comment]
@vaishali-sharma-20 You Forgot to add the Images here too.

**Pull request merged**